tensorboard/webapp/runs/views/runs_table/filterbar_component.ts

```diff
@@ -56,12 +58,18 @@ export class FilterbarComponent {
     return this.filters.get(this.selectedFilterName);
   }
 
+  constructor(private readonly customModal: CustomModal) {}
+
+  ngAfterViewChecked() {
+    this.customModal.runChangeDetection();
```

Comment:
Required to prevent annoying NG0100 error logs in dev. Without this, modal view container change detection will trigger after the host view change detection finishes, potentially causing angular to complain that a expression is changed after it has already been checked.